src/layer/marker/Marker.js

```diff
@@ -36,6 +36,18 @@ export var Marker = Layer.extend({
 		// Whether the marker is draggable with mouse/touch or not.
 		draggable: false,
 
+		// @option autoPan: Boolean = false
+		// Set it to `true` if you want the map to do panning animation when marker hits the edges.
+		autoPan: false,
+
+		// @option autoPanPadding: Point = Point(5, 5)
+		// Equivalent of setting both top left and bottom right autopan padding to the same value.
+		autoPanPadding: [50, 50],
+
+		// @option autoPanSpeed: Number = 100

```

Comment:
Isn't this value responsible about what will be shown in the docs as the default value?
If that is the case then these are wrong for both `autoPanPadding` and `autoPanSpeed`